packages/source-cockpit/README.md

```diff
@@ -0,0 +1,142 @@
+# @gridsome/source-cockpit
+
+> Cockpit source for Gridsome. This package is under development and
+API might change before v1 is released.
+
+## Install
+- `yarn add @gridsome/source-cockpit`
+- `npm install @gridsome/source-cockpit`
+
+## Usage
+
+```js
+module.exports = {
+  plugins: [
+    {
+      use: '@gridsome/source-cockpit',
+      options: {
+        accessToken: process.env.ACCESSTOKEN,
+        host: process.env.APIHOST
+      }
+    }
+  ]
+}
+```
+
+You will need to provide the ACCESSTOKEN and APIHOST environment variables in a `.env` or similar file at the root of your project. Gridsome will automatically read this and pass in the variables to the source plugin.
+
+For example:
+
+```
+APIHOST=https://my-cockpit-cms.com
+ACCESSTOKEN=b90d9080711bab6e9af34fbea754c5
+```
+
+## API limit
+
+You can also set a limit per request if you have thousands of collections or assets and want to pull in a paged set of results. To do this set the `APILIMIT` environment variable, for example:                                      │
```

Comment:
There is a pipe `|` too far from the text.

Reply:
Just a typo, can be removed